cli/flags.rs

```diff
@@ -1643,10 +1643,16 @@ fn compat_arg<'a, 'b>() -> Arg<'a, 'b> {
 fn watch_arg<'a, 'b>() -> Arg<'a, 'b> {
   Arg::with_name("watch")
     .long("watch")
+    .value_name("FILES")
+    .min_values(0)
+    .takes_value(true)
+    .use_delimiter(true)
+    .require_equals(true)
```

Comment:
If `require_equals` isn't set, parsing fails when trying to run without any external files:
`deno run --watch main.js`
It's not clear whether `main.js` is part of `--watch` or if it's the main entry point.
Though this does mean that in `--help` it's not super clear that external files are optional. Because it shows up as `--watch=<FILES>`

Reply:
That makes sense, we already use the same setup in other flags.